db/envstubs.c

```diff
@@ -65,6 +67,9 @@ uint64_t comdb2fastseed(void)
     int seed[2];
     uint64_t out;
 
+    if (external_fastseed)
+        return external_fastseed();
+
     node = gbl_mynodeid;
```

Comment:
My understanding is that `comdb2fastseed` is only needed in the bbenv build. Since you're dropping `fastseed64` into bb-plugins, maybe we could delete the rest of code here?

Reply:
It's also used by local replicants to generate unique record ids, so we're stuck with it for a bit.

Reply:
You mean the seqno? I thought it was just genid.

Reply:
Yes, you're right. It can be configured to use either fastseed or genid.